apps/site/pages/en/learn/typescript/publishing-a-ts-project.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Publishing a TypeScript project
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Publishing a TypeScript project
+
+This article covers items regarding typescript publishing specifically.
```

Comment:
Here we can write a little more about the fact that we're publishing a package that will go on the npm registery.
It's not like an express app that you run. package ≠ app